src/System.Runtime.Extensions/src/Configurations.props

```diff
@@ -1,13 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
```

Comment:
Same here, leave it as it was, you don't need PackageConfigurations for this guy.